dns/dnssec.py

```diff
@@ -15,7 +15,8 @@
 
 """Common DNSSEC-related functions and constants."""
 
-import cStringIO
+import six
+from six import BytesIO
```

Comment:
The `io` module also holds both BytesIO and StringIO (available with >= 2.6). As opposed to six, io is a standard python library, thus future compatible.
(Also applies to many other files in this diff)